firefox-ios/Client/Frontend/Autofill/Address/AddressListViewModel.swift

```diff
@@ -50,6 +50,8 @@ class AddressListViewModel: ObservableObject, FeatureFlaggable {
         return themeManager.getCurrentTheme(for: windowUUID).type == .dark
     }
 
+    let editAddressWebViewManager: WebViewPreloadManaging = EditAddressWebViewManager()
```

Comment:
I think we should pass this at init to have the option to mock it

Reply:
Updated in https://github.com/mozilla-mobile/firefox-ios/pull/20738/commits/64e61ed79fe6e92feefea108da2736f57015c0fc